netlify/edge-functions/mcp-server.ts

```diff
@@ -95,7 +95,13 @@ function formatResponse(data: unknown): {
 }
 
 // Netlify Edge Function Handler
-export default async function handler(req: Request) {
+export default async function handler(req: Request, { url }: Context) {
+  if (req.method !== "POST" && url.pathname === "/mcp") {
+    return Response.redirect(
+      "https://docs.astro.build/en/guides/build-with-ai/",
+      302
+    );
+  }
```

Comment:
Smallest nitpick in the history of mankind
```suggestion
  }

```